demo/app/components/keyboard-control-layer/keyboard-control-layer.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { KeyboardControlService } from '../../../../src/services/keyboard-control/keyboard-control.service';
+import { KeyboardAction } from '../../../../src/models/ac-keyboard-action.enum';
+
+@Component({
+  selector: 'keyboard-control-layer',
+  template: '',
+})
+export class KeyboardControlLayerComponent implements OnInit, OnDestroy {
+  constructor(private keyboardControlService: KeyboardControlService) {
+
+  }
+
+  ngOnInit() {
+    this.keyboardControlService.setKeyboardControls({
+      W: { action: KeyboardAction.CAMERA_FORWARD },
+      S: { action: KeyboardAction.CAMERA_BACKWARD },
+      D: { action: KeyboardAction.CAMERA_RIGHT },
+      A: { action: KeyboardAction.CAMERA_LEFT },
+    }, (keyEvent: KeyboardEvent) => {
```

Comment:
1. how do I register to arrow key? 'arrowUp': {action: ....}
2. the second func if we want to set 2 keys to the same action? we cant just do: 
```
W: {action: LEFT}
L:  {action: LEFT}
```